### PR TITLE
Sdcicd 917 boilerplate update for osde2e

### PR DESCRIPTION
**Pull request by ritmun**
https://issues.redhat.com/browse/SDCICD-917 

Changes

1. deleted osde2e hack folder
2. added osde2e boilerplate subscription
3. Generated harness scaffolding under osde2e/

(Note: Any comments on files under /boilerplate are better made on the openshift/boilerplate repository. Those files can't be modified under consumer repo.)

Description
Using AVO as example, create boilerplate convention to automate osde2e initialization within operator.

Acceptance criteria:
Convention provides make targets to
create osde2e directory, initializes test structure
push docker image of harness to quay repo

### CODE REVIEW

File: osde2e/Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
+    
 ENV PKG=/go/src/github.com/openshift/aws-vpce-operator/
-WORKDIR ${PKG}
-
-# compile test binary
+WORKDIR ${PKG}    
 COPY . .
-
+
```
Comment (dustman9000) [inline osde2e/Dockerfile:8]: not: trailing whitespace

Comment (ritmun) [reply osde2e/Dockerfile:8]: PTAL https://github.com/openshift/boilerplate/pull/272

**Review by mjlshen — APPROVED** [review]
/approve

Looks good to me

**Pull request merged**